server/emails/templates/BaseEmail.tsx

```diff
@@ -9,6 +13,13 @@ import { TaskPriority } from "@server/queues/tasks/BaseTask";
 import { NotificationMetadata } from "@server/types";
 import { getEmailMessageId } from "@server/utils/emails";
 
+export enum EmailMessageCategory {
+  Authentication = "authentication",
```

Comment:
Currently only `Authentication` is used for anything, but while we're here might as well categorize all emails for safety